comp/core/status/statusimpl/status.go

```diff
@@ -193,17 +193,26 @@ func (s *statusImplementation) GetStatus(format string, verbose bool, excludeSec
 			}
 
 			if len(section) > 0 {
+				headerBuffer := new(bytes.Buffer)
+				sectionBuffer := new(bytes.Buffer)
 
 				for i, provider := range s.sortedProvidersBySection[section] {
+
 					if i == 0 {
-						printHeader(b, provider.Section())
-						newLine(b)
+						printHeader(headerBuffer, provider.Section())
+						newLine(headerBuffer)
 					}
-					if err := provider.Text(verbose, b); err != nil {
+					if err := provider.Text(verbose, sectionBuffer); err != nil {
 						errs = append(errs, err)
 					}
 				}
 
+				if sectionBuffer.Len() == 0 {
+					continue
+				}
+
+				b.Write(headerBuffer.Bytes())
+				b.Write(sectionBuffer.Bytes())
 				newLine(b)
```

Comment:
we changed this file because we don't want to print the header of the section if the section is empty

Reply:
LTGM overall however will wait a full time to approve, maybe @GustavoCaso since he know very well the status code

Reply:
@jedupau I'm wondering if you could avoid modifying the code so rather than not displaying the the section header, you could modify the template to display something like `No Parsing Errors`. Similar to how other component status templates do it:

```
{{- with .endpointsInfos -}}
{{ range $key, $value := .}}
  {{$key}} - API Key{{ if gt (len $value) 1}}s{{end}} ending with:
    {{- range $idx, $apikey := $value }}
      - {{$apikey}}
    {{- end}}
  {{- end}}
{{- else }}

No endpoints information. The agent may be misconfigured.
{{end }}
```

Reply:
Hum we thought about this idea but the thing is that most of the agents don't have the snmp integration, so adding a section which is useless for almost everyone seemed weird to us